src/pages/tutorials/create-invite-link.md

```diff
@@ -0,0 +1,313 @@
+## Steps
+
+### 1. Set up the project
+
+#### 1.1. Install dependencies
+
+Install the [MetaMask Delegation Toolkit](https://www.npmjs.com/package/@metamask/delegation-toolkit) and other dependencies in your project:
+
+```bash npm2yarn
+npm install @metamask/delegation-toolkit @web3auth/modal wagmi @tanstack/react-query
+```
+
+#### 1.2. Set up Embedded Wallets (Web3Auth)
+
+Configure [MetaMask Embedded Wallets (Web3Auth)](/embedded-wallets/) to enable users to instantly connect to your dapp using familiar login methods, like social accounts or email.
+
+1. Add a `WEB3AUTH_CLIENT_ID` environment variable, replacing `<YOUR-CLIENT-ID>` with your Web3Auth Client ID:
+
+    ```txt title='.env.local'
+    WEB3AUTH_CLIENT_ID=<YOUR-CLIENT-ID>
+    ```
+
+2. Configure Web3Auth options:
+
+    ```tsx title='providers/AppProvider.tsx'
+    import { WEB3AUTH_NETWORK, Web3AuthOptions } from '@web3auth/modal';
+
+    const web3AuthOptions: Web3AuthOptions = {
+      clientId: process.env.WEB3AUTH_CLIENT_ID as string,
+      web3AuthNetwork: WEB3AUTH_NETWORK.SAPPHIRE_MAINNET,
+    };
+
+    const web3authConfig = {
+      web3AuthOptions,
+    };
+    ```
+
+3. Create a connect button:
+
+    ```tsx title='components/ConnectButton.tsx'
+    import { useWeb3AuthConnect } from '@web3auth/modal/react';
+    import Button from '@/components/Button'; // You can add your own Button component
+
+    export default function ConnectButton() {
+      const { connect } = useWeb3AuthConnect();
+
+      return (
+        <div className='flex gap-2'>
+          <Button onClick={() => connect()}>Connect with Web3Auth</Button>
+        </div>
+      );
+    }
+    ```
+
+#### 1.3. Set up Wagmi
+
+Wrap your dapp with the Wagmi, Web3Auth, and React Query providers.
+Add Wagmi using the Web3Auth Wagmi adapter so wallet connections from Web3Auth are available to Wagmi hooks.
+
+```tsx title='providers/AppProvider.tsx'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { Web3AuthProvider } from '@web3auth/modal/react';
+import { WagmiProvider } from '@web3auth/modal/react/wagmi';
+
+const queryClient = new QueryClient();
+
+export function AppProvider({ children }: { children: React.ReactNode }) {
+  return (
+    <Web3AuthProvider config={web3authConfig}>
+      <QueryClientProvider client={queryClient}>
+        <WagmiProvider>{children}</WagmiProvider>
+      </QueryClientProvider>
+    </Web3AuthProvider>
+  );
+}
+```
+
+### 2. Create Viem clients
+
+#### 2.1. Create a Public Client
+
+Create a [Viem Public Client](https://viem.sh/docs/clients/public) using Viem's `createPublicClient` function.
+You will configure a Bundler Client with the Public Client, which you can use to query the signer's account state and interact with the blockchain network.
+
+```tsx
+import { createPublicClient, http } from 'viem';
+import { sepolia as chain } from 'viem/chains';
+
+const publicClient = createPublicClient({
+  chain,
+  transport: http(),
+});
+```
```

Comment:
I think you also mentioned about getting this from Wagmi which was good point, this can be skipped and we can use wagmi hooks instead.

Reply:
Added in the commit.